xml/depl_maintenance.xml

```diff
@@ -1503,6 +1503,295 @@ migrate</screen>
     the selected nodes.
    </para>
   </sect2>
+  <sect2 xml:id="sec.depl.maintenance.upgrade.troubleshooting">
+   <title>Common Upgrade Problems and Possible Solutions</title>
+    <qandaset>
+     <qandaentry>
+      <question>
+       <para>
+	Upgrade of the admin server has failed.
+       </para>
+      </question>
+      <answer>
+       <para>
+	Check for empty, broken, and not signed repositories in the &admserv;
+	upgrade log file <filename>/var/log/crowbar/admin-server-upgrade.log</filename>. Fix the
+	repository setup. Upgrade then remaining packages manually to &cloudos;
+	and &cloud; &productnumber; using the command <command>zypper dup</command>. Reboot the &admserv;.
+       </para>
+      </answer>
+     </qandaentry>
+     <qandaentry>
+      <question>
+       <para>
+	Upgrade step repeatedly fails due to timeout.
```

Comment:
I would say "Some upgrade step", as there are more of them. But maybe my English is wrong in this case

Reply:
Do you mean something like _a certain upgrade step_?

Reply:
I don't know, is that better? I just mean that there is a number of upgrade steps, and any of them could time out

Reply:
OK. Then I think something simple _An upgrade step_ would work best.